src/sentry/models/groupsubscription.py

```diff
@@ -20,6 +20,7 @@ class GroupSubscriptionReason(object):
     assigned = 2
     bookmark = 3
     status_change = 4
+    setting = 5
```

Comment:
this should be something like deploy_setting or something since its specific to that